src/Divider/Divider.js

```diff
@@ -11,7 +11,7 @@ export const styleSheet = createStyleSheet('MuiDivider', (theme) => {
   return {
     root: {
       height: 1,
-      margin: '0 -1px 0 0',
+      margin: 0,
```

Comment:
It had marginTop -1 in order to not take any space at all. Not sure why this is needed, but its a  change compared to the master version. In other words before that fix it was broken because it was by accident marginLeft -1 instead of marginTop, now its no margin at all. Hope you guys know what you are doing.